govcd/vapp_test.go

```diff
@@ -137,6 +136,19 @@ func (vcd *TestVCD) Test_AddMetadata(check *C) {
 
```

Comment:
I recommend not only test that Task completed successful but and query vapp metadata and verify is there.

Reply:
Fixed.